flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/sql/ExpressionsITCase.scala

```diff
@@ -120,4 +122,26 @@ class ExpressionsITCase(
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
 
+  @Test
+  def testAdvancedDataTypes(): Unit = {
```

Comment:
This is meant to be the unit test for all future advanced data types. It has the same name as the test in `ExpressionITCase` of Scala Table API.